### PR TITLE
Release v1.5.4

### PR DESCRIPTION
**Pull request by freemansw1**
PR for v1.5.4 release 

* [ ] Have you followed our guidelines in CONTRIBUTING.md? 
* [ ] Have you self-reviewed your code and corrected any misspellings? 
* [ ] Have you written documentation that is easy to understand?
* [ ] Have you written descriptive commit messages? 
* [ ] Have you added NumPy docstrings for newly added functions? 
* [ ] Have you formatted your code using black? 
* [ ] If you have introduced a new functionality, have you added adequate unit tests?
* [ ] Have all tests passed in your local clone? 
* [ ] If you have introduced a new functionality, have you added an example notebook?
* [ ] Have you kept your pull request small and limited so that it is easy to review? 
* [ ] Have the newest changes from this branch been merged?

### CODE REVIEW

File: examples/Example_OLR_Tracking_model/Example_OLR_Tracking_model.ipynb
Comment (w-k-jones) [inline examples/Example_OLR_Tracking_model/Example_OLR_Tracking_model.ipynb:1]: This notebook has a `RuntimeError: Requested MovieWriter (ffmpeg) not available` shown in for the video output. Could you try re-running it?

**Review by w-k-jones — COMMENTED** [review]
Thanks for preparing the release! There is one issue I spotted in the rendering of one of the notebooks, but once that is resolved I'm happy for this to be merged and released

File: examples/Example_OLR_Tracking_model/Example_OLR_Tracking_model.ipynb
Comment (freemansw1) [reply examples/Example_OLR_Tracking_model/Example_OLR_Tracking_model.ipynb:1]: Thanks for the catch. Had forgotten to resave that one. Resolved now.

**Review by w-k-jones — APPROVED** [review]

**Review by JuliaKukulies — APPROVED** [review]
Looks all good, thanks for doing the release @freemansw1 !

**Pull request merged**